### PR TITLE
Moduls

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Create a page like the last week's. But, the JS is divided into modules. and install the luxon library and use it to display the time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+>- There is a date and time at the buttom of the page
+
+![screenshot](./img/Screenshot.png)
+
+
+Additional description about the project and its features.
+
+## live demo
+
+- project: (https://eng-mohamed-eldeeb.github.io/Book_list_ES6/)
```
Comment (elmar8287) [inline README.md:20]: - [ ] Please, consider that your demo live link is not working correctly:
Screen Shot 2022-02-22 at 00 35 56

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- <link rel="stylesheet" href="https://moment.github.io/luxon/global/luxon.min.js"> -->
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider removing unused code to make your job clear 👍

File: modules/showFun.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const booksUL = document.querySelector('.book-list');
+export default function show() {
```
Comment (elmar8287) [inline modules/showFun.js]: - [ ] Please, consider using arrow function syntax, as requested in the project 👍 
```
const show() => {
....
}
```

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @eng-mohamed-eldeeb,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @eng-mohamed-eldeeb 👋🏻, 

Good job on making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**